readme.md

```diff
@@ -2,24 +2,28 @@
 
 DISCLAIMER : TEZOS REWARD DISTRIBUTOR IS PROVIDED AS IS. IT IS UNDER CONSTANT DEVELOPMENT. EVENT THOUGH IT IS WELL TESTED, PLEASE USE IT WITH CARE. ALWAYS MAKE A PRE-RUN IN DRY MODE BEFORE MAKING ACTUAL PAYMENTS. IF YOU WANT TO RUN IN SERVICE MODE DO IT AFTER YOU ARE CONFIDENT WITH THE APPLICATION. IN SERVICE MODE ONLY UPDATE IF NEEDED.
 
+PRIVACY : TEZOS REWARD DISTRIBUTOR COLLECTS ANONYMOUS STATISTICS. PLEASE READ OUR [STATISTICS POLICY](https://github.com/tezos-reward-distributor-organization/tezos-reward-distributor/blob/master/docs/statistics.rst) FOR MORE INFORMATION.
+
 ## Tezos Reward Distributor : Run & Forget [![Build Status](https://travis-ci.com/tezos-reward-distributor-organization/tezos-reward-distributor.svg?branch=master)](https://travis-ci.com/tezos-reward-distributor-organization/tezos-reward-distributor)
 
 TRD is a software for distributing staking rewards of delegators introduced in detail in this [Medium article](https://medium.com/@huseyinabanox/tezos-reward-distributor-e6588c4d27e7). This is not a script but a full scale application which can continuously run in the background as a Linux service. It can track cycles and make payments. However it does not have to be used as a service, but it can also be used interactively.
 The documentation can be found [here](https://tezos-reward-distributor-organization.github.io/tezos-reward-distributor/).
 
-TRD supports complex payments, pays in batches, provides three back ends for calculations: Tezos RPC, [tzstats API](https://tzstats.com/) and TzKT API. Developed and tested extensively by the community.
+TRD supports complex payments, pays in batches, and provides three back ends for calculations: Tezos RPC, [tzstats API](https://tzstats.com/) and [TzKT API](https://tzkt.io/). TRD is developed and tested extensively by the community.
 
-**[Powered by TzKT API](https://tzkt.io/)**
```

Comment:
This line cannot be deleted as it is "part of the terms of use"

Reply:
TRD is only powered by TzKt if and when the end-user decides to use it. TRD itself is not "powered by tzkt". Having this line presents a false picture. TzKt is one of several options; it is not the only option.